_data/navigation.js

```diff
@@ -194,7 +194,12 @@ module.exports = {
             title: 'Introduction to Price Feeds',
             url: '/docs/using-chainlink-reference-contracts/',
           },
-          { title: 'Get the Latest Price', url: '/docs/get-the-latest-price/' },
+          { title: 'Get the Latest Price (EVM)',
+            url: '/docs/get-the-latest-price/'
+          },
+          { title: 'Get the Latest Price (Solana)',
+            url: '/docs/get-the-latest-price-solana/'
+          },
```

Comment:
I don't think we should have this page in navigation as there are a lot of not EVM-compatible chains.

Reply:
We can keep it simple for now, but we'll look at restructuring the content later.